README.md

```diff
@@ -1,73 +1,73 @@
+1. User clicks "Connect your car" button (or similar button) on your application's website.
+2. User is redirected to an OAuth authentication page, using the `openDialog`, `addClickHandler`, or `generateLink` methods. This page requires the user to authenticate with their vehicle credentials.
+3. After entering their credentials, the user will be presented with a list of their vehicles to authorize and a selection of permissions to grant your app.
+4. Smartcar's services will redirect back to your application's redirect URI. If the user approves the authorization, the redirect URI will include a `code` query parameter holding the authorization code. If they deny access, the URI will include `error` and `error_description` parameters. If you source `redirect.js` on your redirect page, it will trigger execution of your optionally provided `onComplete` method (passed to the `Smartcar` constructor on the client) and close out the redirect page.
 
-    *Note:* in the Smartcar-hosted scheme `redirect.js` handles extracting the query parameters from the redirect URI. In a manual hosting scheme you can choose to do this extraction client side (making use of `redirect.js` if you'd like) or server side. If they approved access you'll need to exchange the `code` parameter for an access token. If they denied access you'll need to handle the error (passed in the `error` argument).
-5. Your application's backend server will need to accept the authorization code and exchange it for an access token.
+    *Note:* in the Smartcar-hosted scheme, `redirect.js` handles extracting the query parameters from the redirect URI. In a manual hosting scheme, you can choose to do this extraction client side (making use of `redirect.js` if you'd like) or the server side. If the user approves access, you'll need to exchange the `code` parameter for an access token. If they deny access, you'll need to handle the error (passed as the first argument).
+5. Your application's back end server will need to accept the authorization code and exchange it for an access token.
 
 ## Next Steps
 
-This SDK facilitates OAuth link generation, click handler creation and popup dialog creation. This SDK will not assist in exchanging authorization codes for an access token. Should you choose to self-host the redirect this SDK will not assist with extracting the authorization code query parameter from the redirect URI.
+This SDK facilitates OAuth link generation, click handler creation, and pop-up dialog creation. This SDK will not assist in exchanging authorization codes for an access token. If you choose to self-host the redirect, this SDK provdes a script you can source on the redirect that will assist you with extracting the authorization code query parameter.
```

Comment:
this SDK **provides** a **script, which** you can source on the **redirect. The script** will assist you **in** extracting the authorization code query parameter.